crates/python_utilities/src/ast_printer.rs

```diff
@@ -154,7 +164,8 @@ fn emit_body<'a>(body: &'a [Stmt], str_buffer: &mut WritingBuffer<'a>) {
             }
 
             Stmt::Expr(ast::StmtExpr { range: _, value }) => {
-                str_buffer.push(Cow::Owned(format!("{}", value)));
+                let expr: &ast::Expr = &*value;
+                CustomDisplay::custom_fmt(expr, str_buffer, false);
```

Comment:
can we do `expr.custom_fmt(str_buffer, false);` instead?

actually, I think `value.custom_fmt(str_buffer, false);` could be made to work...

Reply:
Addressed in 64150be9f93ffc954a02cfebda6f65460d294c17